src/App.js

```diff
@@ -26,6 +27,7 @@ const App = () => {
         <Route path="/faq" component={FAQPage} />
         <Route exact path="/projects/:projectId" component={ProjectSelectPage} />
         <Route path="/projects/:projectId/work" component={ProjectWorkPage} />
+        <Route path="/" component={DefaultPage} />
```

Comment:
I'd remove the `path` prop here as it should not be needed and could be misunderstood to be the route for the home page. I'd also rename "DefaultPage" to "NotFoundPage" or something to that effect for the same reason. 

https://reactrouter.com/web/example/no-match